dds/DCPS/XTypes/DynamicDataImpl.cpp

```diff
@@ -5962,6 +6000,640 @@ bool operator<<(Serializer& ser, const KeyOnly<DDS::DynamicData_ptr>& key)
   return serialize(ser, key.value, Sample::KeyOnly);
 }
 
+} // namespace DCPS
+
+namespace DCPS {
```

Comment:
```suggestion
} // namespace DCPS

namespace DCPS {
```

Reply:
I don't see the difference from the suggested change, but I assume the suggestion is to merge these two `namespace DCPS` blocks?

Reply:
Yes, that's what I meant